private/buf/cmd/buf/command/alpha/protoc/protoc.go

```diff
@@ -192,6 +194,10 @@ func run(
 			}
 			span.End()
 		}
+		wasmPluginExecutor, err := bufwasm.NewPluginExecutor(filepath.Join(container.CacheDirPath(), "wasmplugins"))
```

Comment:
It feels like this cache path should be a const somewhere, and not in an individual command's code - @pkwarren thoughts on where this should go?

Reply:
Also perhaps just named "wasm" or "wasmplugin"